blocks/form/form.js

```diff
@@ -0,0 +1,217 @@
+function createSelect(fd) {
```

Comment:
Please add a comment to this file indicating its purpose & use.

Reply:
Removed this method.